deploy.py

```diff
@@ -861,6 +866,14 @@ def main(args):
             "storage (i.e. unplugging the key will reset the key)."),
   )
 
+  main_parser.add_argument(
+      "--elf2tab-output",
+      metavar="FILE",
+      dest="elf2tab_output",
+      default=None,
```

Comment:
I would specify this argument with a type:
```python
main_parser.add_argument(
  "--elf2tab-output",
  metavar="FILE",
  type=argparse.FileType("a"),
  dest="elf2tab_output",
  default=None,
```

This way, opening/closing the file is left to the argparse module and you don't have to handle errors.
The code above that logs the error will then be:
```python
if self.args.elf2tab_output:
  self.args.elf2tab_output.write(self.checked_command_output(elf2tab_args))
else:
  self.checked_command(elf2tab_args)
```